lib/cli.rb

```diff
@@ -787,6 +787,25 @@ def self.supports_filtering(command)
     end
   end
 
+  if @config.experimental_features
+    desc "Dockerize a system description"
+    long_desc <<-LONGDESC
+      Detects workloads from a system description and creates docker containers.
```

Comment:
I would say "... and creates a recommendation for a corresponding docker container setup".